### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_AddMessage_ScreenReader:When we edit the expire field and visibility timeout field and set a value that leads to appear of error message, the message is not read by the NVDA.

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5724]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_Azure Tools Storage Explorer_May2022; #Azure Tools Storage Explorer; #WCAG4.2.2; #ScreenReader; #NVDA; #FTP;

## Actual results:
When we edit the "Expire in" and "Visibility timeout" field, and a error message appears at the bottom of the window, same message is not read by the NVDA.
For example: If "Expire in" field set to 8 and the dropdown present beside the field set to "Days", an error appears at the bottom of the screen -"The message's time to live must be no less than 1 second and no more than 7 days". The appeared message is not read by the NVDA.

## Expected results:
If an error appears, it should be announced at that instant..

## Environmental Details:
Application: Microsoft Azure Storage Explore
version: 1.24.0

## OS Details
Microsoft Windows 11 Enterprise
version 21H2 OS Build 22000.613

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Open Explorer" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to "test" and click on it.
7. Navigate to "AddMessage" button and click on it.
8. Turn on the NVDA.
9. Edit "Expire in"/"Visibility Timeout" field and set a value that will leads to the appear of error message at the bottom of the pane.
10. Check whether NVDA is announcing the error message after modifying the fields.

## User Impact:
Error message should be announced instantly when appears on the screen.

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Attachments

https://user-images.githubusercontent.com/105198219/168733747-dbf17db7-218e-47c3-9342-fb7af179e334.mp4

**Comment by MRayermannMSFT** [comment]
@ManishC1 I cannot repro this bug. Can you let me know/reconfirm:
- What Storage Explorer version you are using?
- What version of NVDA you are using?

**Comment by ManishaC1** [comment]
@MRayermannMSFT 

- Storage Explorer version is 1.24.0
- NVDA version is 2021.3.5

**Comment by ManishaC1** [comment]
Issue is resolved in the provided environment (ver 1.25.0). Hence closing the bug.
![5724_Issue not repro](https://user-images.githubusercontent.com/105198219/175875688-78f56283-4ef8-4454-bc47-cc90b080f98e.png)

**Comment by ManishaC1** [comment]
Github tags:#Closed

**Comment by Vishnuvarthancr7** [comment]
GitHubTags:#Closed;
